src/Hl7.Fhir.Support.Tests/Model/ConceptTests.cs

```diff
@@ -6,7 +6,10 @@
  * available at https://raw.githubusercontent.com/FirelyTeam/firely-net-sdk/master/LICENSE
  */
 
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Support.Tests.Utils;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
 using P = Hl7.Fhir.ElementModel.Types;
 
 namespace Hl7.FhirPath.Tests
```

Comment:
You did not change the namespace, but I am very surprised that the namespace is here `Hl7.FhirPath.Tests` . Should we not change that?

Reply:
Huh? Yeah, why not, it's just a test, we can change it!